README.md

```diff
@@ -262,11 +262,11 @@ types:
   passed to Trino as a time with a time zone
 * the result of `trino.Timestamp(year, month, day, hour, minute, second,
   nanosecond)` - passed to Trino as a timestamp without a time zone
+* `time.Duration` - passed to Trino as an interval day to second (precision can be lost for very large durations)
```

Comment:
Trino day to hour interval has thes limitations:
- Max precision is in milliseconds (for duration it's nano)
- Max int precision is int32
   - This has no impact on day, hour or minute intervals since this is plenty (time.Duration is int64 but when devided by time.Hour or time.Minute the value is always < maxInt32)
   - For seconds it could be for large durations that duration/time.Second > maxInt32 (this is the case when duration is around 68 years)
 - For second duration there is a limit of 10 digits when there is a dot (so 123_456_789.1 and 12_345_678.91 seconds are valid but 12_345_678.912 is not)
    - Worse case: interval of 10_000_000.001 seconds is the smallest duration that is not valid (around 116 days)
  
We can return error in these case but since we round to millis anyway (otherwise most duration would cause errors) it think the durations are large enough that you don't care about that precision anyway

Reply:
Users should be able to round the values themselves, and we should check if the nanos part is zero, otherwise return an error. This is how Trino handles all temporal values.